### PR TITLE
RHDEVDOCS 5935 Add pipelinerun and builder artifacts to chains config

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

please cp to pipelines-docs-1.14

Issue:

RHDEVDOCS 5935

Link to docs preview:

https://74503--ocpdocs-pr.netlify.app/openshift-pipelines/latest/secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.html#supported-parameters-tekton-chains-configuration_using-tekton-chains-for-openshift-pipelines-supply-chain-security

see Table 2 and Table 5

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mramendi** [comment]
https://issues.redhat.com/browse/RHDEVDOCS-5935

File: modules/op-supported-parameters-tekton-chains-configuration.adoc [hunk 1]
```diff
@@ -61,6 +61,11 @@ Cluster administrators can use various supported parameter keys and values to co
 | The signature backend for signing pipeline run payloads.
 | `x509`, `kms`
 | `x509`
+
+| `artifacts.pipelinerun.enable-deep-inspection`
+| When this parameter is `true`, {tekton-chains} records the results of the child task runs of a pipeline run. When this parameter is `false`, {tekton-chains} records the results of the pipeline run but not of its child task runs.
+| `x509`, `kms`
```
Comment (PuneetPunamiya) [inline modules/op-supported-parameters-tekton-chains-configuration.adoc:67]:
```suggestion
| `"true", "false"`
```

For ref: https://github.com/tektoncd/chains/blob/main/docs/config.md#pipelinerun-configuration

File: modules/op-supported-parameters-tekton-chains-configuration.adoc [hunk 2]
```diff
@@ -61,6 +61,11 @@ Cluster administrators can use various supported parameter keys and values to co
...
+| `x509`
```
Comment (PuneetPunamiya) [inline modules/op-supported-parameters-tekton-chains-configuration.adoc:68]:
```suggestion
| `"false"`
```

File: modules/op-supported-parameters-tekton-chains-configuration.adoc [hunk 3]
```diff
@@ -132,6 +137,12 @@ Cluster administrators can use various supported parameter keys and values to co
 |
 | `+https://tekton.dev/chains/v2+`
 
+| `builddefinition.buildtype`
+| The build type for in-toto attestation. When this parameter is `\https://tekton.dev/chains/v2/slsa`, {tekton-chains} records in-toto attestations in strict conformance with the SLSA v1.0 specification. When this parameter is `\https://tekton.dev/chains/v2/slsa-tekton`, {tekton-chains} records in-toto attestations with additional information such as the labels and annotations in each task run and pipeline run, and also adds each task in a pipeline run under `resolvedDependencies`.
```
Comment (PuneetPunamiya) [inline modules/op-supported-parameters-tekton-chains-configuration.adoc:141]:
```suggestion
| The build type for in-toto attestation. When this parameter is `\https://tekton.dev/chains/v2/slsa`, {tekton-chains} records in-toto attestations in strict conformance with the SLSA v1.0 specification. When this parameter is `\https://tekton.dev/chains/v2/slsa-tekton`, {tekton-chains} records in-toto attestations with additional information such as the labels and annotations in each taskrun and pipelinerun, and also adds each pipeline task in a pipelinerun under `resolvedDependencies`.
```

**Comment by PuneetPunamiya** [comment]
LGTM

**Review by ppitonak — APPROVED** [review]

File: modules/op-supported-parameters-tekton-chains-configuration.adoc [hunk 4]
```diff
@@ -61,6 +61,11 @@ Cluster administrators can use various supported parameter keys and values to co
 | The signature backend for signing pipeline run payloads.
 | `x509`, `kms`
 | `x509`
+
+| `artifacts.pipelinerun.enable-deep-inspection`
+| When this parameter is `true`, {tekton-chains} records the results of the child task runs of a pipeline run. When this parameter is `false`, {tekton-chains} records the results of the pipeline run but not of its child task runs.
```
Comment (lpettyjo) [inline modules/op-supported-parameters-tekton-chains-configuration.adoc:66]: Add a comma before "but".

File: modules/op-supported-parameters-tekton-chains-configuration.adoc [hunk 5]
```diff
@@ -132,6 +137,12 @@ Cluster administrators can use various supported parameter keys and values to co
 |
 | `+https://tekton.dev/chains/v2+`
 
+| `builddefinition.buildtype`
+| The build type for in-toto attestation. When this parameter is `\https://tekton.dev/chains/v2/slsa`, {tekton-chains} records in-toto attestations in strict conformance with the SLSA v1.0 specification. When this parameter is `\https://tekton.dev/chains/v2/slsa-tekton`, {tekton-chains} records in-toto attestations with additional information such as the labels and annotations in each `TaskRun` and `PipelineRun` object, and also adds each task in a `PipelineRun` object under `resolvedDependencies`.
```
Comment (lpettyjo) [inline modules/op-supported-parameters-tekton-chains-configuration.adoc:141]: Add command before "such".

**Review by lpettyjo — COMMENTED** [review]
Two small comments. Otherwise, LGTM!

**Pull request merged**